### PR TITLE
Chrome extension fix: users should still be able to create link when prompted for feedback

### PR DESCRIPTION
**Pull request by natashapetrus**
https://www.pivotaltracker.com/story/show/176200084

### CODE REVIEW

File: ChromeExtension/groupsReview.html [hunk 1]
```diff
@@ -32,48 +32,42 @@
...
-
-            <center>
-                <p style="margin-bottom: 5px;" id="result">&nbsp;</p>
-            </center>
-
-            <div id="dialog" title="Error">
-                <div id="networkError"></div>
-            </div>
-        </div>
+        <center>
```
Comment (ssundheim) [inline ChromeExtension/groupsReview.html:35]: While we are in here, mind replacing these HTML4 `<center>` tags? Eg, with `<div style="text-align: center;">`

File: ChromeExtension/js/background.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ document.addEventListener('DOMContentLoaded', function () {
 
     } else {
         chrome.browserAction.setPopup({
-            popup: "groups.html"
+            popup: "groupsReview.html" // todo: change back to groups.html, only for testing
```
Comment (skazn) [inline ChromeExtension/js/background.js:47]: Don't forget to change this back.

**Review by ssundheim — APPROVED** [review]
Looks good from an HTML standpoint. Just one optional request.

File: ChromeExtension/groupsReview.html [hunk 3]
```diff
@@ -32,48 +32,42 @@
...
+            <button class="btn-gl-blue" style="width: 250px; font-size: 13px; " data-bind="event: { click: createLinkFromButton }">New link from current page</button>
```
Comment (skazn) [inline ChromeExtension/groupsReview.html:36]: These styles should probably exist where ever the css for this button is ya?

File: ChromeExtension/groupsReview.html [hunk 4]
```diff
@@ -32,48 +32,42 @@
...
+            <p class="fine-print" style="position: relative; top: 8px;">Or, right-click a link and select "Create..."</p>
```
Comment (skazn) [inline ChromeExtension/groupsReview.html:37]: Same with this style

**Review by skazn — CHANGES_REQUESTED** [review]
@natashapetrus are those versioned zip files supposed to be checked in? I'm guessing you delete the old one and check in the new one, should they both be gitignored?

File: ChromeExtension/js/background.js (hunk 2 above)
Comment (natashapetrus) [reply ChromeExtension/js/background.js]: done

File: ChromeExtension/groupsReview.html (hunk 1 above)
Comment (natashapetrus) [reply ChromeExtension/groupsReview.html]: done

Comment (ssundheim) [reply ChromeExtension/groupsReview.html]: Thanks!

**Pull request merged**